common/app/common/configuration.scala

```diff
@@ -19,52 +19,65 @@ import scala.util.{Failure, Success, Try}
 
-object InstallVars {
+object Environment {
 
-  val installVars = new File("/etc/gu/install_vars") match {
+  private[this] val properties = {
+    def env(name: String): Option[String] = sys.env.get(name)
+
+    val props = new File("/etc/gu/install_vars") match {
       case f if f.exists => IOUtils.toString(new FileInputStream(f), Charset.defaultCharset())
       case _ => ""
     }
 
-  val properties = Properties(installVars)
+    for {
+      p <- Properties(props)
+    } yield p match {
+      case (key, value) => key -> env(key).getOrElse(value)
+    }
+  }
 
-  def apply(key: String, default: String): String = properties.getOrElse(key, default)
+  val stack = properties.getOrElse("stack", "frontend")
+  val app = properties.getOrElse("app", "dev-build")
+  val stage = properties.getOrElse("STAGE", "DEV")
+  val awsRegion = properties.getOrElse("region", "eu-west-1")
+  val configBucket = properties.getOrElse("configBucket", "aws-frontend-store")
 
-  object InstallationVars {
-    val stack = apply("stack", "frontend")
-    // if got config at app startup, we wouldn't need to configure it
-    val app = apply("app", "dev-build")
-    val stage = apply("STAGE", "DEV")
-    val awsRegion = apply("region", "eu-west-1")
-    val configBucket = apply("configBucket", "aws-frontend-store")
-    if (stage == "DEV" && new File(s"${System.getProperty("user.home")}/.gu/frontend.properties").exists) {
+  if (stage == "DEV" && new File(s"${System.getProperty("user.home")}/.gu/frontend.properties").exists) {
     throw new RuntimeException(
-      "\n\nYou have a file ~/.gu/frontend.properties with secrets - please delete that file and any copies as it is not needed.\n  " +
-      "All secrets are now stored in S3 bucket aws-frontend-store, not on your laptop.\n\n  " +
-      "Should you need to override any properties in DEV, create a new file ~/.gu/frontend.conf. \n" +
-      "For an example see https://github.com/guardian/frontend/blob/master/common/app/common/configuration.scala#L48\n" +
-      "For details of the changes see https://github.com/guardian/frontend/pull/14081")
-    /*
-    ~/.gu/frontend.conf example file:
-
-    # local development (DEV stage) config overrides (not secrets)
-    devOverrides {
-      switches.key=DEV/config/switches-yournamehere.properties
-      facia.stage=CODE
-    }
-
-     */
-    }
+      """
+        |You have a file ~/.gu/frontend.properties with secrets - please delete that file and any copies as it is not needed.
+        |
+        |All secrets are now stored in S3 bucket aws-frontend-store, not on your laptop.
+        |
+        |Should you need to override any properties in DEV, create a new file ~/.gu/frontend.conf.
+        |
+        |For an example see https://github.com/guardian/frontend/blob/master/common/app/common/configuration.scala#L48
+        |
+        |For details of the changes see https://github.com/guardian/frontend/pull/14081
+      """.stripMargin
+    )
```

Comment:
Do we need to keep this? This is from more than one year ago I don't think anybody has a `frontend.properties` file anymore

Reply:
I did wonder that. Happy to delete if we think it's okay!